templates/crud_rest_api_dartfrog/routes/repos/[repo].dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:io';
+
+import 'package:collection/collection.dart';
+import 'package:crud_rest_api_dartfrog/users.dart';
+import 'package:dart_frog/dart_frog.dart';
+
+import 'index.dart';
+
+Future<Response> onRequest(RequestContext context, String id) async {
+  final request = context.request;
+  final repoId = int.tryParse(id);
+  if (repoId == null) {
+    return Response(statusCode: HttpStatus.badRequest);
+  }
```

Comment:
this looks like it should be a part of middleware, but I'm not sure if dart_frog allows per-route middlewares

Reply:
https://github.com/invertase/globe/blob/frog_rest_api_example/templates/rest_api/routes/todos/%5Bid%5D/_middleware.dart

Reply:
Resolved.